core/com.b2international.snowowl.core/src/com/b2international/snowowl/core/console/CodeSystemsCommand.java

```diff
@@ -81,21 +81,77 @@ public void run(CommandLineStream out) {
 	}
 
 	private String getCodeSystemInfo(CodeSystem codeSystem) {
+		//Added the three spaces for better readability
 		return new StringBuilder()
-			.append("Name: ").append(codeSystem.getName()).append("\n")
-			.append("Short name: ").append(codeSystem.getShortName()).append("\n")
-			.append("OID: ").append(codeSystem.getOid()).append("\n")
-			.append("Maintaining organization link: ").append(codeSystem.getOrganizationLink()).append("\n")
-			.append("Language: ").append(codeSystem.getPrimaryLanguage()).append("\n")
-			.append("Repository: ").append(codeSystem.getRepositoryId()).append("\n")
-			.append("Working branch: ").append(codeSystem.getBranchPath()).append("\n")
-			.append("Terminology ID: ").append(codeSystem.getTerminologyId())
-			.append(showVersions ? getCodeSystemVersionsInfo(codeSystem) : "")
+			.append(codeSystem.getShortName()).append("\n")
+			.append("\tName: ").append(codeSystem.getName()).append("\n")
+			.append("\tShort name: ").append(codeSystem.getShortName()).append("\n")
+			.append("\tOID: ").append(codeSystem.getOid()).append("\n")
+			.append("\tMaintaining organization link: ").append(codeSystem.getOrganizationLink()).append("\n")
+			.append(codeSystem.getCitation() == null? "" :
+					"\tCitation: " + codeSystem.getCitation() + "\n")
+			.append(getAvailableUpgradesInfo(codeSystem))
+			.append("\tCode System URI: ").append(codeSystem.getCodeSystemURI()).append("\n")
+			.append("\tIcon Path: ").append(codeSystem.getIconPath()).append("\n")
+			.append(getDependenciesInfo(codeSystem))
+			.append(codeSystem.getUpgradeOf() == null? "" : 
+					"\tUpgrade of: " + codeSystem.getUpgradeOf().toString() + "\n")
+			.append(codeSystem.getExtensionOf() == null? "" : 
+					"\tExtension of: " + codeSystem.getExtensionOf().toString() + "\n")
+			.append("\tRepository: ").append(codeSystem.getRepositoryId()).append("\n")
+			.append("\tWorking branch: ").append(codeSystem.getBranchPath()).append("\n")
+			.append("\tTerminology ID: ").append(codeSystem.getTerminologyId())
+			.append(getAdditionalPropertiesInfo(codeSystem))
+			.append(getLocalesInfo(codeSystem))
+			.append(showVersions ? "\t" + getCodeSystemVersionsInfo(codeSystem) : "")
 			.toString();
 	}
 	
+	private String getAdditionalPropertiesInfo(CodeSystem codeSystem) {
+		if(codeSystem.getAdditionalProperties() == null || codeSystem.getAdditionalProperties().isEmpty()){
+			return "";
+		}
+		
+		StringBuilder result = new StringBuilder("\n\tAdditional Properties:\n");
+		codeSystem.getAdditionalProperties().forEach((key, value) -> result.append("\t\t").append(key).append(": ").append(value.toString()).append("\n"));;
+		
+		return result.toString();
+	}
+
+	private String getDependenciesInfo(CodeSystem codeSystem) {
+		if(codeSystem.getDependencies() == null || codeSystem.getDependencies().isEmpty()){
```

Comment:
These "dependencies" are only tooling dependencies and only certain extension plug-ins are using it. Feel free to remove it.

Reply:
Done. Should I remove outputs related to repository info too?